zbus/src/address/transport/tcp.rs

```diff
@@ -55,22 +55,22 @@ impl Tcp {
         self
     }
 
-    /// Returns the `tcp:` address `host` value.
+    /// Return the `tcp:` address `host` value.
```

Comment:
Actually "Return" is redundant here so you might as well remove that.

Reply:
Done